org.freedesktop.fldigi.metainfo.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2020 hamradio lu9dce -->
+<component type="desktop-application">
+ <id>org.freedesktop.fldigi</id>
```

Comment:
is this really the app-id? e.g. is this actually part of the freedesktop.org project?

Reply:
I really can't answer this to you .. where did I get that information from? or what corresponds to fence?